src/main/java/edu/umich/tl/Utils.java

```diff
@@ -64,6 +66,13 @@ public static InputStream getInputStreamFromURL(String fileURL) {
 
 		return in;
 	}
+	public static String dateChopper(String dateAsString) {
```

Comment:
Also, you might want to look into using [joda-time](http://www.joda.org/joda-time/).  Anthony has used it and likes it.